keycloak/themes/uid2-theme/email/messages/messages_en.properties

```diff
@@ -1,7 +1,8 @@
 emailVerificationBody=Please verify your email address\n\nClick the link below to verify this email and complete your account creation for Unified ID.\n\n{0}
 passwordResetSubject=Reset password
-passwordResetBodyHtml=<div style="text-align: center;"> <p style="text-align: left;"> Hey {0},<br /><br /> Forgot your Password?<br><br> We received a request to reset your password for your account.<br><br> Please use the button below to reset your password. </p> </div> <div style="margin: 30px 0"> <a href="{1}" style=" border-radius: 5px; font-weight: 700; font-size: 18px; padding: 15px 120px 15px 120px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Reset Password</a > </div>
+passwordResetBodyHtml=<div style="text-align: center"> <p style="text-align: left"> Hey {0},<br /><br /> Forgot your Password?<br /><br /> We received a request to reset your password for your account.<br /><br /> Please use the button below to reset your password. </p> </div> <div style="margin: 66px 0"> <a href="{1}" style=" border-radius: 5px; font-weight: 700; font-size: 18px; padding: 10px 23px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Reset Password</a > </div>
 passwordResetBody=<p>Hey {0}, Forgot your Password? We received a request to reset your password for your account. Please use the link below to reset your password.{1}</p>
-inviteBodyHtml=<div style="text-align: center"> <p style="text-align: left"> Hey {0},<br /><br /> You&rsquo;ve been added to a Unified ID Portal team.<br /><br /> Please use the button below to create a password for your account. </p> </div> <div style="margin: 67px 0"> <a href="{1}" style=" border-radius: 5px; font-weight: 700; font-size: 18px; padding: 15px 120px 15px 120px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Accept Invitation</a > </div>
-inviteBody=<p>Hey {0}, You&rsquo;ve been added to Unified ID Portal.. Please use the link below to create a password for your account.{1}</p>
\ No newline at end of file
+inviteBodyHtml=<div style="text-align: center"> <p style="text-align: left"> Hey {0},<br /><br /> You've been added to the UID2 Portal.<br /><br /> Please use the button below to create a password for your account. </p> </div> <div style="margin: 67px 0"> <a href="{1}" style=" border-radius: 5px; font-weight: 600; font-size: 18px; padding: 10px 19px; margin: 0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center; text-decoration: none; " >Accept Invitation</a > </div>
```

Comment:
We cannot get the name of person who send the invite, so I removed the `by {someone}`